tests/utils/test_schemapi.py

```diff
@@ -648,6 +633,11 @@ def chart_error_example__four_errors():
     )
 
 
+# NOTE: These 6 `TypeError`s are due to the removal of `kwargs` in Chart.encode
+# That is, only declared keywords can now be used in Chart.encode,
+# so these are not conditions worth checking against.
```

Comment:
I believe this NOTE is only relevant for the PR review so it can be removed from the code.

Reply:
Fixed in [`0e0e167` (#3444)](https://github.com/vega/altair/pull/3444/commits/0e0e167e11b298dd971707dd2f41d2ad5e54ad16)